src/PeriodEnd/CoInvestedPayments/DeployCoInvestedPayments/sql/dml/PeriodEnd.CoInvestedPayments.Cleanup.Deds.DML.sql

```diff
@@ -1,11 +1,11 @@
+DECLARE @CollectionPeriodName varchar(8) = (SELECT [AcademicYear] FROM Reference.CollectionPeriods WHERE [Open] = 1) + '-' + (SELECT [Name] FROM Reference.CollectionPeriods WHERE [Open] = 1)
+
```

Comment:
Do we not need the ${DAS_PeriodEnd.FQ} token? And if not, why have it in the other SQL statements? Also is this just a tidy up as this code as it existed should work.

Reply:
The script is executed on the transient database and this table exists on the transient database